src/solver/utils/mps_utils.cpp

```diff
@@ -320,49 +324,60 @@ void OPT_EcrireJeuDeDonneesLineaireAuFormatMPS(void* Prob, int optNumber, uint n
 // Full mps writing
 // --------------------
 fullMPSwriter::fullMPSwriter(PROBLEME_SIMPLEXE_NOMME* named_splx_problem,
-                             int optNumber,
-                             uint thread_number) :
- named_splx_problem_(named_splx_problem),
- current_optim_number_(optNumber),
- thread_number_(thread_number)
-{
-}
+                             int year,
+                             int week,
+                             int optNumber) :
+    I_MPS_writer(year, week, optNumber),
+    named_splx_problem_(named_splx_problem)
+{}
+
 void fullMPSwriter::runIfNeeded(Solver::IResultWriter::Ptr writer)
 {
-    OPT_EcrireJeuDeDonneesLineaireAuFormatMPS(
-      (void*)named_splx_problem_, current_optim_number_, thread_number_, writer);
+    OPT_EcrireJeuDeDonneesLineaireAuFormatMPS((void*)named_splx_problem_,
+                                              year_,
+                                              week_,
+                                              current_optim_number_,
+                                              writer);
 }
 
 // ---------------------------------
 // Full mps writing by or-tools
 // ---------------------------------
-fullOrToolsMPSwriter::fullOrToolsMPSwriter(MPSolver* solver, int optNumber, uint thread_number) :
- solver_(solver), current_optim_number_(optNumber), thread_number_(thread_number)
+fullOrToolsMPSwriter::fullOrToolsMPSwriter(MPSolver* solver,
+                                           int year,
+                                           int week,
+                                           int optNumber) :
+    I_MPS_writer(year, week, optNumber),
+    solver_(solver)
 {
 }
 void fullOrToolsMPSwriter::runIfNeeded(Solver::IResultWriter::Ptr writer)
 {
     // Make or-tools print the MPS files leads to a crash !
```

Comment:
This comment is now irrelevant, please remove it

Reply:
Question : from which version of or-tools was the problem fixed ?

Reply:
Done